### PR TITLE
Remove obsolete warnings for object type decision + object cast

### PR DESCRIPTION
**Pull request by Robinfr**
Starting from Mendix 10.5, these will also be available in nanoflows.

### CODE REVIEW

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/cast-object.md [hunk 1]
```diff
@@ -5,10 +5,6 @@ weight: 10
 tags: ["studio pro", "cast object", "activity"]
 ---
 
-{{% alert color="warning" %}}
-This activity can only be used in **Microflows**.
-{{% /alert %}}
-
 ## 1 Introduction
 
 The **Cast object** activity is used in a microflow after an [object type decision](/refguide/object-type-decision/) to change the type of object from the generalized object type to the specialized object type of the path out of the object type decision.
```
Comment (EnasAbdelrazek) [inline content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/cast-object.md]: Here we can mention nanoflow as well.

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/decisions/object-type-decision.md [hunk 2]
```diff
@@ -9,10 +9,6 @@ aliases:
 #If moving or renaming this doc file, implement a temporary redirect and let the respective team know they should update the URL in the product. See Mapping to Products for more details.
 ---
 
-{{% alert color="info" %}}
-This activity can only be used in microflows, not in nanoflows.
-{{% /alert %}}
-
 ## 1 Introduction
 
 An object type decision is an element that makes a choice based on the type of an object of a generalized entity. The output of the object type decision are the specialized entities that inherit from the generalized entity. For more information on specialization and generalization, see [Entities](/refguide/entities/).
```
Comment (EnasAbdelrazek) [inline content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/decisions/object-type-decision.md:14]: We can also mention nanoflow in `If you want to use the specialized type in the rest of the microflow you can use a [Cast](/refguide/cast-object/) activity.`

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/cast-object.md (hunk 1 above)
Comment (Robinfr) [reply content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/object-activities/cast-object.md]: I've removed the word microflow here.

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/decisions/object-type-decision.md (hunk 2 above)
Comment (Robinfr) [reply content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/decisions/object-type-decision.md:14]: I've added the word nanoflow here, as it made more sense than removing it.

**Review by EnasAbdelrazek — APPROVED** [review]

**Review by Yiyun333 — APPROVED** [review]

**Pull request merged**